pom.xml

```diff
@@ -22,7 +22,7 @@
     </parent>
 
     <properties>
-      <java.version>1.8</java.version>
```

Comment:
Not sure why we are reducing to java 1.7 ... do we need to test all the modules and make sure nothing breaks?